_posts/2018/2018-06-24-ignore-php-bots-with-rack-attack.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Ignore PHP bots with Rack::Attack
+description: 'This is not the wp_login.php you’re looking for.'
+layout: article
+category: ruby
+image:
+  base: '2018/ignore-php-bots-with-rack-attack'
+  alt: 'Robots'
+  source: 'https://unsplash.com/photos/xdEeLyK4iBo'
+  credit: 'Jehyun Sung'
+---
+
+If your site has been around for a long time or starts getting popular, you might see a bunch of 404 errors in your logs.
+
+Sometimes these errors are for pages like `/wp_login.php` or other PHP files. If you see these, it's likely automated bots scanning the internet for security vulnerabilities in WordPress.
```

Comment:
*it's likely to _be_ automated bots scanning the _Internet_ for security vulnerabilities in WordPress _sites_